docs/GDI-SOP_ISM.md

```diff
@@ -0,0 +1,176 @@
+
+## 6 Introduction and Background Information
+This document provides the necessary procedures to manage information effectively across all activities involved in the delivery and management of GDI services at both the European and national node levels. It emphasises the importance of preserving the confidentiality, integrity, and accessibility of information as outlined in the GDI SOP collection.
+
+Standard Operating Procedures (SOPs) are vital to maintaining uniformity and high standards across the GDI infrastructure. They ensure consistent practices, fostering better interoperability, security, and an improved user experience. To maintain these standards, it is crucial to develop clear procedures for Information Service Management (ISM) to guide the implementation of SOPs, ensuring that all staff adhere to best practices and contribute to the optimal functioning of the GDI infrastructure.
+
+
+## 7 Information Service Management Procedures
+
+### 7.1 GDI SOP creation
+
+#### 7.1.1 European-level SOPs
+
+For each European-level GDI SOP, only one SOP document is produced which all GDI partners/nodes are expected to follow. The need for a new European-level GDI SOP is first identified by one or more GDI project partners who will then prepare a proposal, making use of the GitHub repository "[New SOP Request](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/issues/new/choose)" GH issue. The proposal should include the purpose and scope of the SOP and a justification for its need. Once the proposal for a new European-level GDI SOP is approved by the OC/SDPC, then work can begin on creating the new SOP following the Steps below (Figure 1). For more in detail information, refer to this "[Context Diagram.](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/dev/sops/european-level/GDI-SOP0007_SOP-template-creation.md#7-summary-or-context-diagram)"
+
+**Steps:**
+
+1. After a SOP request is received in the repository, a OC/SCDP member evaluates the request. If it is a valid request,the OC/SCDP member prepares the SOP draft from the [GDI General SOP template.](./GDI-SOP_sop-template.md)
+2. OC/SCDP member shares template with the authors( OC/SDPC or nominated experts) to write the SOP (i.e. fill in the content)
+3. After the drafting is completed, the OC/SCDP member shares the draft internally or with experts to review for completeness.
+4. Both committees (OC/SDPC) approve the reviewed document.
+5. The OC/SDPC member then shares the approved SOP with GDI MB for authorization.
+6. GDI MB reviews the SOP and authorise or requests changes (repeat from Step 3)
+7. OC/SDPC accessions ([7.2](#72-gdi-sop-accessioning-and-file-namin)) the authorised SOP according to agreed process and the SOP goes into production
+8. OC/SDPC initiates periodic review cycle (repeat Steps 3-6)
+
+![Valid: Proposal for creating and approving European-level SOPs.](images/GDI-ISM_schema_European_Level.png)

```

Comment:
Another comment on the picture is that the last two bits, cloning and reviewing, sound to me like the OC/SDPC reviews the "instances" from the nodes, rather than the SOP "instance" that lives in the main repository.

I would probably modify the diagram last bit, since the review occurs in parallel with the cloning, and is not dependent on it.